docs/source/stats-collection.rst

```diff
@@ -0,0 +1,72 @@
+.. _stats_collection:
+
+Comparing Spider Executions
+===========================
+
+Sometimes it is worthy to compare results from previous executions of your
+spider. For example, we should be able to track whether the number of items
+returned is lower than previous executions, which may indicate a problem.
+
+Scrapy allows you to collect stats in the form key/value, but these stats are
+lost when the spider finishes its execution. Spidermon provides a built-in
+stats collector that stores the stats of all spider executions in your local
+file system.
+
+After enabling this stats collector, every spider instance running will have a
+``stats_history`` attribute containing a list of the latest spider executions
+that can be easily accessed in your monitors.
+
+To enable it, include the following code in your project settings:
+
+.. code-block:: python
+
+    # myproject/settings.py
+    STATS_CLASS = (
+        "spidermon.contrib.stats.statscollectors.LocalStorageStatsHistoryCollector"
+    )
+
+    # Stores the stats of the last 10 spider execution (default=100)
+    SPIDERMON_MAX_STORED_STATS = 10
+
+The following example shows how we can verify whether the number of items
+returned in the actual spider execution is higher than 90% of the mean of items
+returned in the latest spider executions.
```

Comment:
Instead of ‘actual’ and ‘latest’, I would talk about ‘current’ and ‘previous’.

Although ‘actual’ (en) _can_ mean ‘actual’ (pt, es), it is usually a false friend meaning ‘real’ (pt, es). In general, ‘actual’ (pt, es) should be translated as ‘current’ (en).

Here ‘latest’ could be confused with actual/current, so ‘previous’ may be a better choice.

Reply:
Much better. Thanks!